src/main/java/io/vertx/ext/mail/impl/Capabilities.java

```diff
@@ -88,6 +100,9 @@ void parseCapabilities(final String message) {
       if (c.equals("STARTTLS")) {
         capaStartTLS = true;
       }
+      if (c.equals("PIPELINING")) {
```

Comment:
how about using a switch on string here to simplify code ?

Reply:
There are also checks on `startsWith()` for `AUTH ` and `SIZE `, which makes this not appropriate to use `switch`